src/parser/clauses/return.c

```diff
@@ -8,13 +8,11 @@
 #include "../../util/arr.h"
 #include "../../util/rmalloc.h"
 
-AST_ReturnElementNode* New_AST_ReturnElementNode(AST_ArithmeticExpressionNode *exp, const char* alias) {
+AST_ReturnElementNode* New_AST_ReturnElementNode(AST_ArithmeticExpressionNode *exp, char* alias) {
 	AST_ReturnElementNode *returnElementNode = (AST_ReturnElementNode*)malloc(sizeof(AST_ReturnElementNode));
 	returnElementNode->exp = exp;
-	returnElementNode->alias = NULL;
+	returnElementNode->alias = alias ? strdup(alias) : NULL;
```

Comment:
Why are we duplicating alias?
Looking at the grammar:
```
returnElement(A) ::= arithmetic_expression(B) AS UQSTRING(C). {
	A = New_AST_ReturnElementNode(B, C.strval);
}
```
It seems like it is already duplicated.

Reply:
Good call! The logic in `ExpandCollapsedNodes` calls this function and was not duplicating alias, which was causing some errors, but memory should be duplicated there rather than here.